current-reviews-data.ipynb

```diff
@@ -897,17 +952,378 @@
+       "      <th>2023Q4</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2024Q2</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>xmnlab</th>\n",
+       "      <th>2021Q4</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>yeelauren</th>\n",
+       "      <th>2023Q3</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>zeitsperre</th>\n",
+       "      <th>2024Q1</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                            n_edits\n",
+       "editor                          created_at         \n",
+       "Batalex                         2024Q1            1\n",
+       "cmarmo                          2024Q2            1\n",
+       "ctb                             2024Q1            1\n",
+       "dhomeier                        2024Q1            1\n",
+       "hamogu                          2024Q2            2\n",
+       "isabelizimm                     2024Q2            1\n",
+       "lwasser                         2019Q1            1\n",
+       "                                2019Q4            1\n",
+       "snacktavish Emily Jane McTavish 2023Q3            1\n",
+       "sneakers-the-rat                2023Q4            1\n",
+       "                                2024Q2            1\n",
+       "xmnlab                          2021Q4            1\n",
+       "yeelauren                       2023Q3            1\n",
+       "zeitsperre                      2024Q1            1"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "n_edits = count_edits_by_quarter(reviews_df)[\"n_edits\"].to_frame()\n",
+    "for editor in ignore_editors:\n",
+    "    n_edits.drop(editor, inplace=True)\n",
+    "n_edits"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2024-07-14T23:34:19.359304Z",
+     "iopub.status.busy": "2024-07-14T23:34:19.359109Z",
+     "iopub.status.idle": "2024-07-14T23:34:19.364809Z",
+     "shell.execute_reply": "2024-07-14T23:34:19.364050Z",
+     "shell.execute_reply.started": "2024-07-14T23:34:19.359285Z"
+    },
+    "tags": [
+     "remove-input"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Get axes min and max values so all plots have the same scale\n",
+    "\n",
+    "# Get vmin and vmax for the dates\n",
+    "min_date = reviews_df.created_at.min().to_period(\"Q\").start_time\n",
+    "end_of_this_quarter = pd.Timestamp.today().to_period(\"Q\").end_time\n",
+    "time_domain = (min_date, end_of_this_quarter)\n",
```

Comment:
Right now the ticks have a .5 increment. can we please make that increment 1 instead?

<img width="719" alt="Screenshot 2024-08-09 at 7 20 24 PM" src="https://github.com/user-attachments/assets/41987120-d608-4ae4-9a8f-8b864be418dc">